projects/ion/src/lib/sidebar/sidebar.component.ts

```diff
@@ -41,10 +42,12 @@ export class IonSidebarComponent {
 
   public itemSelected(itemIndex: number): void {
     selectItemByIndex(this.items, itemIndex);
+    this.closeOnSelect ? this.toggleSidebarVisibility() : false;
```

Comment:
You not need a ternary here.
```suggestion
    if (this.closeOnSelect ) {
      this.toggleSidebarVisibility()
    }
```